File: 프로그래머스/1/131705. 삼총사/삼총사.js
function solution(number) {
    var answer = 0;
    for(let i = 0 ; i<number.length-2 ; i++){
        for(let j = i+1 ; j<number.length-1 ; j++){
            for(let k = j+1 ; k<number.length ; k++){
               let arr =[number[i] , number[j] , number[k]]; 
               let sum = arr.reduce((item , cur) => item+cur , 0) ;
               sum === 0 ? answer++ : answer ;  
            }
        }
    }
    return answer;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    my_list =[]
    count = 0
    for k in range(len(number)-2) :
        for s in range(k+1, len(number)-1) :
            for t in range(s+1, len(number)) :
                num = (number[k] , number[s] , number[t])
                if sum(num) == 0:
                    count += 1
    return count 